core/src/chacha_cuda.rs

```diff
@@ -162,7 +163,7 @@ mod tests {
         let entries = make_tiny_test_entries(32);
         let ledger_dir = "test_encrypt_file_many_keys_multiple";
         let ledger_path = get_tmp_ledger_path(ledger_dir);
-        let ticks_per_slot = 16;
+        let ticks_per_slot = DEFAULT_TICKS_PER_SLOT;
```

Comment:
this is a test-specific value, shouldn't be affected by the global one.

Reply:
These tests are failing even without this change. I changed this hoping it'll fix the test. Let me revert this and push a patch.